Tests/conf.json

```diff
@@ -2,6 +2,11 @@
     "testTimeout": 160,
     "testInterval": 20,
     "tests": [
+        {
+            "integrations": "URLhaus",
```

Comment:
This is still in the build. please check this.
<img width="1071" alt="Continuous Integration and Deployment 2020-04-28 10-30-33" src="https://user-images.githubusercontent.com/19407287/80459654-5af06080-893b-11ea-86c0-337ff60351cf.png">